packages/core/database/lib/dialects/mysql/schema-inspector.js

```diff
@@ -229,6 +232,19 @@ class MysqlSchemaInspector {
 
     return Object.values(ret);
   }
+
+  async getDatabaseInformation() {
+    const [results] = await this.db.connection.raw(SQL_QUERIES.VERSION);
+    const version = results[0]['version()'];
+
+    const [versionNumber, databaseName] = version.split('-');
+    const database = databaseName && databaseName.toLowerCase() === 'mariadb' ? 'MariaDB' : 'MySQL';
```

Comment:
You could use constants for the database names to avoid string checks afterwards.
It would be nice to add this in the dialect API instead and get the version info in the init phase of the dialect. 

That would avoid doing this in the schema inspector and make the API cleaner